packages/frontend-v2/src/components/molecules/paymentCard.tsx

```diff
@@ -0,0 +1,98 @@
+import React, { useEffect, useState } from 'react';
+import { IMembership } from '../../libs/interfaces/membership.interface';
+import { IPayment } from '../../libs/interfaces/payment.interface';
+import { planTypeMapper } from '../../services/data-mapping.service';
+import { membershipService } from '../../services/membership.service';
+import { paymentService } from '../../services/payment.service';
+import { paymentRequestService } from '../../services/paymentRequest.service';
+
+interface IProps {
+  memberShip: IMembership;
+  onPaymentReceivedFunction: CallableFunction;
```

Comment:
onPaymentReceiveFunction -> onPaymentReceived (we already know it is a function). And change the type of CallableFunction by ()=>void